Api/IGlamourerApiItems.cs

```diff
@@ -58,4 +58,22 @@ public interface IGlamourerApiItems
     /// Prefer to use the index-based function unless you need to get the state of someone currently unavailable.
     /// </remarks>
     public GlamourerApiEc SetBonusItemName(string playerName, ApiBonusSlot slot, ulong bonusItemId, uint key, ApplyFlag flags);
+
+    /// <summary> Set the defined Meta State flags to the active or inactive state on actor. </summary>
+    /// <param name="objectIndex"> The game object index of the actor to be manipulated. </param>
+    /// <param name="types"> The flags defining which Meta State's to update to the new value. </param>
+    /// <param name="newValue"> The new value (true or false) to update to. </param>
+    /// <param name="key"> A key to unlock or lock the state if necessary. </param>
+    /// <param name="flags"> The flags used for the reversion. Respects Once (see <see cref="ApplyFlag"/>.)</param>
+    /// <returns> ItemInvalid, ActorNotFound, ActorNotHuman, InvalidKey, Success. </returns>
+    public GlamourerApiEc SetMeta(int objectIndex, SetMetaFlag types, bool newValue, uint key, ApplyFlag flags)

```

Comment:
I'm fixing those myself, so don't worry about it, but: missed a semicolon here and below

Reply:
They also don't really belong into Items... but I'll keep them there because they vaguely are associated to items and State has a lot more functions already.